src/main/java/com/employeesService/EmployeesService/repository/JdbcDepartmentRepository.java

```diff
@@ -0,0 +1,68 @@
+package com.employeesService.EmployeesService.repository;
+
+import com.employeesService.EmployeesService.model.Department;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.*;
+
+@Repository
+public class JdbcDepartmentRepository implements DepartmentRepository {
+
+    private JdbcTemplate jdbcTemplate;
```

Comment:
When we do constructor injection, dependencies should be marked as `final`, so please do that on all dependencies on the project